liqd_product/templates/500.html

```diff
@@ -1,8 +1,56 @@
-{% extends "base_errors.html" %}
-{% load i18n %}
+{% load i18n static %}
+<html>
 
-{% block error_status %}500{% endblock %}
+<head>
+    <title>Error &mdash; {% trans 'Beteiligung.in' %}</title>
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'platform.css' %}" />
+</head>
 
-{% block error_description %}
-    <p>{% trans 'Internal server error.' %}</p>
-{% endblock %}
+<body>
+    <div class="l-outer-wrapper">
+        <a href="#main" class="sr-only sr-only-focusable">Skip to content </a>
+        <header class="main-header" aria-label="Header">
+            <div class="main-header__border-top main-header__border-top--platform"></div>
+            <div class="l-wrapper">
+                <div class="platform-brand main-header__brand">
+                    <a href="/" class="platform-brand__link">Beteiligung.in</a>
+                </div>
+            </div>
+        </header>
+            <main id="main">
+                <div class="l-wrapper">
+                    <div class="l-center-6">
+                        <h1>500</h1>
+                        <p>{% trans 'Internal server error.' %}</p>
+                            <a href="/">{% trans 'Back to Beteiligung.in' %}</a>
+                    </div>
+                </div>
+            </main>
+            <footer class="footer">
+                <div class="l-wrapper">
```

Comment:
I think we do not actually need the content of the footer. Otherwise it's hard to keep updated with the actual include for the footer.